src/debugpy/_vendored/pydevd/pydevd.py

```diff
@@ -791,10 +791,10 @@ def collect_try_except_info(self, code_obj):
                     max_line = -1
                     min_line = sys.maxsize
                     for _, line in dis.findlinestarts(code_obj):
-                        if line > max_line:
+                        if line is not None and line > max_line:
```

Comment:
I'm confused about how `line` can be `None` since `dis.findlinestarts` checks for `None` before yielding.

Reply:
I'm not sure what makes you say that? The docs at least say it can be none. dis.findlinestarts is supposed to call: https://docs.python.org/3/reference/datamodel.html#codeobject.co_lines.

I didn't check the source in CPython though

Reply:
Oh, I didn't notice that it wasn't our code. I just used "go to def"` and saw this. Turns out this is from my CPython 3.12.7 install.

```python
def findlinestarts(code):
    """Find the offsets in a byte code which are start of lines in the source.

    Generate pairs (offset, lineno)
    """
    lastline = None
    for start, end, line in code.co_lines():
        if line is not None and line != lastline:
            lastline = line
            yield start, line
    return
```